composer/algorithms/augmix/README.md

```diff
@@ -0,0 +1,108 @@
+# 🎨 AugMix
+
+[\[How to Use\]](#how-to-use) - [\[Suggested Hyperparameters\]](#suggested-hyperparameters) - [\[Technical Details\]](#technical-details) - [\[Attribution\]](#attribution)
+
+`Computer Vision`
+
+For each data sample, AugMix creates an _augmentation chain_ by sampling `depth` image augmentations from a set (e.g. translation, shear, contrast).
+It then applies these augmentations sequentially with randomly sampled intensity.
+This is repeated `width` times in parallel to create `width` different augmented images.
+The augmented images are then combined via a random convex combination to yield a single augmented image, which is in turn combined via a random convex combination sampled from a Beta(`alpha`, `alpha`) distribution with the original image.
+
+| ![AugMix](aug_mix.png) |
+|:--:
+|*An image of a turtle that undergoes three different augmentation chains, gets combined back together using a convex combination, and gets combined with the original image. [Figure 4 from Hendrycks et al. (2020)](https://arxiv.org/abs/1912.02781).*|
+
+## How to Use
+
+### Functional Interface
+
+TODO(MATTHEW): FIX
+
+```python
+def training_loop(model, train_loader):
+  opt = torch.optim.Adam(model.parameters())
+  loss_fn = F.cross_entropy
+  model.train()
+  
+  for epoch in range(num_epochs):
+      for X, y in train_loader:
+          y_hat = model(X)
+          loss = loss_fn(y_hat, y)
+          loss.backward()
+          opt.step()
+          opt.zero_grad()
+```
+
+### Composer Trainer
+
+TODO(MATTHEW): Fix and provide commentary and/or comments
+
+```python
+from composer.algorithms import XXX
+from composer.trainer import Trainer
+
+trainer = Trainer(model="",
+                  train_dataloader=train_dataloader,
+                  max_duration='1ep',
+                  algorithms=[AugMix(severity=3, width=3, depth=-1, alpha=1.0, augmentation_set="all")])
+
+trainer.fit()
+```
+
+### Implementation Details
```

Comment:
```suggestion
## Implementation Details
```

Reply:
Nah, it's part of the code samples, at least for now.